lib/indico.js

```diff
@@ -60,11 +64,12 @@ var make_request = function (name, data, batch, config) {
 
   body = addKeywordArguments(body, config);
   apis = config ? config['apis'] : false
+  version = config ? config['version'] : false
```

Comment:
Smart being able to grab version from config! I didn't see that in python; do we want it with other languages?